careers/ladder.md

```diff
@@ -582,89 +582,212 @@ of their team considerably more than the direct impact of the code they write.
+          <li>Can grow Tech Leads to have a multiplier effect on their team.</li>
+        </ul>
+      </td>
+      <td class="border-top">
+        <ul>
+          <li>Can navigate and lead teams through ambiguous situations, tackle complex issues, and align strategy with company priorities.</li>
+          <li>Significantly accelerates the delivery and increases the impact of multiple teams.</li>
+          <li>Guides teams to provide maximum, user-focused impact. Pushes team to demonstrate greater sense of urgency. Constantly experiments with ways to become more efficient.</li>
+          <li>Regularly drives engineering wide improvements.</li>
+        </ul>
+      </td>
+      <td class="border-top">
+        <ul>
+          <li>Develops strong collaboration with PDDE Leadership team members.</li>
+          <li>Can successfully influence XFN partners to drive improvements across teams (and even more so the Tech Leads).</li>
+          <li>Suggest optimal resource allocation given business goals and constraints.</li>
+          <li>Knows how to combine technical guidance, delivery management and coaching to increase teams' impact.</li>
+          <li>Proactively identifies issues across multiple teams and effectively addresses them.</li>
+        </ul>
+      </td>
+      <td class="border-top">
+        <ul>
+          <li>Get supports from their manager to establish the right growth path for their EM reports, autonomously drives career of all their ICs reports.</li>
+        </ul>
+      </td>
+    </tr>
+    <tr>
+      <td class="border-top">M5</td>
+      <td class="border-top">Director of Engineering</td>
+      <td class="border-top">
+        <ul>
+          <li>Manage and coach Engineering Managers to be effective communicators and managers of their people.</li>
+          <li>Can drive and improve EM hiring pipeline.</li>
+          <li>Ensures that EMs and TLs are mentoring and managing teams effectively towards results.</li>
+        </ul>
+      </td>
+      <td class="border-top">
+        <ul>
+          <li>Proactively identifies and drives engineering wide improvements.</li>
+          <li>Proactively identifies issues across teams and disciplines and effectively addresses them.</li>
+          <li>Drives execution of large & complex PDDE initiatives.</li>
+          <li>Is accountable for long term technical roadmap to support the business needs and unlock growth.</li>
+        </ul>
+      </td>
+      <td class="border-top">
+        <ul>
+          <li>Drives PDDE changes in close partnership with PDDE Leadership team.</li>
+          <li>Actively participates in cross-org leadership activities.</li>
+          <li>Leverages network across the company to develop strong empathy for the various business needs.</li>
+          <li>Can give effective feedback on and influence company strategy.</li>
+          <li>Regularly delegates and increases own impact by helping others to achieve more.</li>
```

Comment:
I would expect this to be true from M3 up

Reply:
Good observation thanks! What I actually mean here is that at M3 I'd expect this to happen at the team level, and at M5, I'd expect that to happen beyond their team and across a large chunk of the business / company. Let me think of a better way to phrase this / clarify.